core/src/main/java/de/danoeh/antennapod/core/preferences/PlaybackSpeedHelper.java

```diff
@@ -0,0 +1,41 @@
+package de.danoeh.antennapod.core.preferences;
+
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.feed.FeedItem;
+import de.danoeh.antennapod.core.feed.FeedMedia;
+import de.danoeh.antennapod.core.feed.MediaType;
+import de.danoeh.antennapod.core.util.playback.Playable;
+
+import static de.danoeh.antennapod.core.feed.FeedPreferences.SPEED_USE_GLOBAL;
+
+public class PlaybackSpeedHelper {
+
+    /**
+     * Returns the currently configured playback speed for the specified media.
+     */
+    public static float getCurrentPlaybackSpeed(Playable media) {
```

Comment:
Moving `getCurrentPlaybackSpeed()` to `FeedMedia` (with a default implementation in `Playable` interface) would be more in-line with the existing pattern. E.g., `FeedMedia.hasAlmostEnded()` also lookups `UserPreference`: 

https://github.com/AntennaPod/AntennaPod/blob/d7f0f95adaa2adf15d3231994dab2eb469d8dd14/core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java#L235-L238

Reply:
I don't like the code duplication that would entail. Feel to change this after merge.